### PR TITLE
Create appimagelint.appdata.xml

### PR DESCRIPTION
**Pull request by probonopd**

### CODE REVIEW

**Comment by TheAssassin** [comment]
Can you please add the deployment logic to the build script?

File: ci/build-appimage.sh [hunk 1]
```diff
@@ -45,6 +45,8 @@ export PIP_WORKDIR="$REPO_ROOT"
 export OUTPUT=appimagelint-x86_64.AppImage
 export VERSION="$SETUPPY_VERSION-git$COMMIT"
 
+mkdir -p AppDir/usr/share/metainfo/ && cp $REPO_ROOT/resources/appimagelint.appdata.xml AppDir/usr/share/metainfo/
```
Comment (TheAssassin) [inline ci/build-appimage.sh]:
```suggestion
mkdir -p AppDir/usr/share/metainfo/ && cp "$REPO_ROOT"/resources/appimagelint.appdata.xml AppDir/usr/share/metainfo/
```

Unsichere Evaluation. Variablen bitte immer in " wrappen (so "eng" wie möglich).

File: resources/appimagelint.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	<name>appimagelint</name>
+	<summary>Check AppImage files for common issues</summary>
+	<description>
+		<p>Check AppImages for compatibility, best practices etc. Powerful functionality combined with simple usage and human-friendly feedback. </p>
+	</description>
+	<launchable type="desktop-id">appimagelint.desktop</launchable>
+	<url type="homepage">https://github.com/TheAssassin/appimagelint/</url>
+	<screenshots>
+		<screenshot type="default">
+			<image>https://user-images.githubusercontent.com/2480569/59196947-977a6880-8b7f-11e9-907e-642090b67ae8.png</image>
```
Comment (TheAssassin) [inline resources/appimagelint.appdata.xml]: Blocked by https://github.com/TheAssassin/appimagelint/pull/8#issuecomment-500415016.

File: ci/build-appimage.sh [hunk 3]
```diff
@@ -45,6 +45,8 @@ export PIP_WORKDIR="$REPO_ROOT"
 export OUTPUT=appimagelint-x86_64.AppImage
 export VERSION="$SETUPPY_VERSION-git$COMMIT"
 
+mkdir -p AppDir/usr/share/metainfo/ && cp "$REPO_ROOT"/resources/appimagelint.appdata.xml AppDir/usr/share/metainfo/
```
Comment (TheAssassin) [inline ci/build-appimage.sh:48]: Not required for now, but do you know `install`? You could've just `install .../...xml AppDir/usr/share/metainfo`, it'll create all the intermediate directories, you can also spare some `chmod/chown` stuff by making use of optional parameters.
Also `install` can serve as a replacement for `mkdir` using `install -d my/dir`, also allows for changing owners or permissions with the same flags a normal call supports.

**Comment by TheAssassin** [comment]
Thanks. Always appreciate not to have to deal with weird AppStream stuff.

File: ci/build-appimage.sh (hunk 3 above)
Comment (probonopd) [reply ci/build-appimage.sh:48]: I like simple tools that everyone understands. But yes, I should learn how to use `install`.

**Comment by probonopd** [comment]
http://output.jsbin.com/qoqukof is taming the monster ;-)

File: ci/build-appimage.sh (hunk 3 above)
Comment (TheAssassin) [reply ci/build-appimage.sh:48]: `install` is awesome, as it saves you typing work. I thought *you* might be really interested in that therefore. It's part of the GNU coreutils, so it's available nearly anywhere.

**Pull request merged**